iotdb-core/ainode/pom.xml

```diff
@@ -56,81 +56,73 @@
                             <includes>
-                                <include>LICENSE</include>
+                                <include>conf/</include>
+                                <include>thrift/</include>
                             </includes>
                         </fileset>
                         <fileset>
                             <directory>target</directory>
-                            <excludes>
-                                <exclude>**/*ainode*</exclude>
-                            </excludes>
                         </fileset>
                         <fileset>
-                            <directory>dist</directory>
+                            <directory>venv</directory>
                         </fileset>
                     </filesets>
                 </configuration>
             </plugin>
-            <!-- generate git.properties to provide build info -->
             <plugin>
-                <groupId>pl.project13.maven</groupId>
-                <artifactId>git-commit-id-plugin</artifactId>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
                 <configuration>
-                    <generateGitPropertiesFile>true</generateGitPropertiesFile>
-                    <generateGitPropertiesFilename>${project.basedir}/iotdb/conf/git.properties</generateGitPropertiesFilename>
-                    <includeOnlyProperties>
-                        <includeOnlyProperty>^git.commit.id.abbrev$</includeOnlyProperty>
-                        <includeOnlyProperty>^git.dirty$</includeOnlyProperty>
-                    </includeOnlyProperties>
-                    <commitIdGenerationMode>full</commitIdGenerationMode>
-                    <failOnNoGitDirectory>false</failOnNoGitDirectory>
-                    <offline>true</offline>
-                    <gitDescribe>
-                        <dirty>-dev</dirty>
-                    </gitDescribe>
+                    <skip>true</skip>
                 </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.gmaven</groupId>
+                <artifactId>groovy-maven-plugin</artifactId>
                 <executions>
+                    <!--
+                      The python versioning follows slightly different rules than Maven.
+                      This little script makes sure the python project file is always in
+                      sync with the maven module.
+                    -->
                     <execution>
+                        <id>sync-python-version</id>
+                        <phase>validate</phase>
                         <goals>
-                            <goal>revision</goal>
+                            <goal>execute</goal>
                         </goals>
+                        <configuration>
+                            <source>resources/syncPythonVersion.groovy</source>
```

Comment:
```suggestion
                            <source>${basedir}/resources/syncPythonVersion.groovy</source>
```

Reply:
Yeah ... that was the other thing I found ;-) Should already be fixed, if you update.